File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None):
    # find the difference between the current size and the minimum size
    padding = len(array) - min_size
    # if the current size is less than the minimum size
    if padding < 0:
        while padding:
            # add padding to the answer array
            array.append(value)
            padding+=1
    
    return array

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, value=null) => {
    let padding = array.length - minSize

    if (padding < 0){
        while(padding){
            array.push(value)
            padding += 1
        }
    }
    return array
}

console.log(pad([1,2,3], 5)) // should == [1,2,3,null,null]
console.log(pad([1,2,3], 5, 'apple')) // should == [1,2,3,'apple', 'apple']